kolibri/plugins/management/assets/src/device_management/views/device-info-page/index.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+
+  <subpage-container>
+    <auth-message v-if="!isSuperuser" authorizedRole="superuser" />
```

Comment:
I had made this comment on the original design: https://github.com/learningequality/kolibri/issues/1910#issuecomment-339171823

Reply:
correct link? that's jessica's comment

Reply:
I edited her comment :)

Reply:
oh i see your edit